test/feed.test.js

```diff
@@ -91,5 +122,47 @@ describe('Post data class tests', () => {
       await feed.setDelayed(60);
       expect(feed.isDelayed()).resolves.toBe(true);
     });
+
+    test('Updated feed should be different from data', async () => {
+      const feed = new Feed(data.author, data.url, data.user, data.link, null, null);
+      feed.author = 'Author Post';
+      feed.url = 'https://modified.user.feed.com/feed.rss';
+      await feed.update();
+      const modifiedFeed = await Feed.byId(feed.id);
+      // modifiedFeed.id should be different from feed.id as feed hasn't been reassigned to updated feed.id value
+      expect(modifiedFeed.id).not.toBe(feed.id);
+      expect(modifiedFeed.id).not.toBe(createFeed.id);
+      expect(modifiedFeed.url).not.toBe(createFeed.url);
+      expect(modifiedFeed.author).not.toBe(createFeed.author);
+      expect(modifiedFeed.url).toBe(feed.url);
+      expect(modifiedFeed.author).toBe(feed.author);
+      expect(modifiedFeed.id).toBe(urlToId(feed.url));
+    });
+
+    test('Removing feeds should also remove posts', async () => {
+      const feed = new Feed(data.author, data.url, data.user, data.link, null, null);
+      await feed.save();
+      postData1.feed = feed.id;
+      postData2.feed = feed.id;
+      await Promise.all([addPost(postData1), addPost(postData2)]);
```

Comment:
Instead of using `addPost`, can we use `new Post()` and `post.save()`?

Same thing below, with `getPost`.  We either need to use the storage methods or the higher order data methods, but shouldn't mix.  I think we should avoid using storage directly as much as possible, since it returns data in different formats that we expect when working with Feeds and Posts.

Reply:
Fixed, I'll be using `Feed.create()` and `Post.create()` instead. Wasn't sure if I should've used the Post class in here since we were testing Feed stuff